src/COBREXA.jl

```diff
@@ -17,6 +17,7 @@ include("builders/core.jl")
 include("builders/genes.jl")
 include("builders/objectives.jl")
 
+include("analysis/modifications/optimizer.jl")
```

Comment:
we should rename this a bit, something like solver settings or so

Reply:
will do